tests/JWT.Tests/project.json

```diff
@@ -0,0 +1,38 @@
+{
+    "version": "1.0.0-*",
+    "testRunner": "xunit",
+    "dependencies": {
+        "dotnet-test-xunit": "2.2.0-preview2-build1029",
+        "xunit": "2.2.0-beta2-build3300",
+        "FluentAssertions": "4.14.0",
+        "JWT": "1.3.7-*"
+    },
+    "frameworks": {
+        /*"net451": {
```

Comment:
xunit only supports >= net451.   I commented this because I was getting errors when attempting to run the tests.   I will try track down the dotnet issue I was reading about this which might give some more context.